runtime/doc/nvim_terminal_emulator.txt

```diff
@@ -325,11 +325,10 @@ This works slightly differently:
   Go back to editing the gdb command with any command that starts Insert mode,
   such as `a` or `i`.
 - The program being debugged will run in a separate window.  On MS-Windows
-  this is a new console window.  On Unix, if the |+terminal| feature is
-  available a Terminal window will be opened to run the debugged program in.
+  this is a new console window.
```

Comment:
this should be reduced to the case of +terminal being available:  `a :terminal window will be opened to run the debugged program in.`